docs/csharp/quick-starts/interpolated-strings-local.md

```diff
@@ -60,49 +73,46 @@ public class Vegetable
    public override string ToString() => Name;
 }
 
-public class Example
+public class Program
 {
-   public enum Unit { item, pound, ounce, dozen };
-   
    public static void Main()
    {
       var item = new Vegetable("eggplant");
       var date = DateTime.Now;
       var price = 1.99m;
-      var unit = Unit.item;
-      Console.WriteLine($"On {date}, the price of {item} was {price} per {unit}.");
+      Console.WriteLine($"On {date}, the price of {item} was {price} per item.");
    }
 }
 ```
-    
-Note that the second interpolated expression includes the `item` object in the result string that's displayed to the console, and in this case the string "eggplant" is inserted into the result string. That's because, when the type of an interpolated expression is not a string, the C# compiler does the following:
 
-- If the interpolated expression is `null`, the interpolated expression returns an empty string ("", or <xref:System.String.Empty?displayProperty=nameWithType>).
+Note that the interpolated expression `item` in the interpolated string resolves to the text "eggplant" in the result string. That's because, when the type of the expression result is not a string, the result is resolved to a string in the following way:
+
+- If the interpolated expression evaluates to `null`, an empty string ("", or <xref:System.String.Empty?displayProperty=nameWithType>) is used.
 
-- If the interpolated expression is not `null`, the `ToString` method of the type of the interpolated expression is called. You can test this by commenting out the definition of the `Vegetable.ToString` method in the example by putting a comment symbol (`//`) in front of it. In the output, the string "eggplant" is replaced by the type name, "Vegetable", which is the default behavior of the <xref:System.Object.ToString?displayProperty=nameWithType> method.   
+- If the interpolated expression doesn't evaluate to `null`, typically the `ToString` method of the result type is called. You can test this by updating the implementation of the `Vegetable.ToString` method. You might even not implement `ToString` method since every C# data type has some implementation of this method. To test that, comment out the definition of the `Vegetable.ToString` method in the example (to do that, put a comment symbol `//` in front of it). In the output, the string "eggplant" is replaced by the fully qualified type name ("Vegetable" in this example), which is the default behavior of the <xref:System.Object.ToString?displayProperty=nameWithType> method.
```

Comment:
An additional sentence here describing the default behavior for enum fields would be useful.